pages/learn/react/index.mdx

```diff
@@ -0,0 +1,4 @@
+# Let's Learn React
```

Comment:
this is the `/learn/react` page, which is interesting demonstration of our ability to override specific "tags" and topics very easily.

Reply:
damn, this breaks the build 😭

Any idea why @johnlindquist ?